pkg/kubectl/cmd/exec.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"io"
+	"os"
+	"strings"
+
+	"github.com/docker/docker/pkg/term"
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+)
+
+func (f *Factory) NewCmdExec(cmdIn io.ReadCloser, cmdOut, cmdErr io.WriteCloser) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "exec <pod> <container> <command> [<args...>]",
```

Comment:
I'm fine with flags. WRT --, are you saying you want it to be like this? `kubectl exec -p pod -c container -- command arg1 arg2 arg...` ?